qutebrowser/commands/runners.py

```diff
@@ -50,18 +50,36 @@ def _current_url(tabbed_browser):
 
 def replace_variables(win_id, arglist):
     """Utility function to replace variables like {url} in a list of args."""
+    tabbed_browser = objreg.get('tabbed-browser', scope='window',
+                                window=win_id)
+    url = _current_url(tabbed_browser)
```

Comment:
This makes some tests fail and is problematic because now with *any* command having a valid URL is mandatory, while before, that was only the case for commands with replacements.

You'll either need to have this call for every variable, or find some other way to not require a valid URL for commands which don't use those replacements.